### ISSUE TITLE
Java 8 incompatability on doctor-plugin 0.7.2 

### ISSUE THREAD

**Issue by kelvinharron** [runningcode/gradle-doctor#171]
Hey Nelson! We're unable to bump Gradle to 7 in our project, is there any way we can stay on new releases with doctor in the meantime? 

I can get you a full stack trace for the below if it's any use.

Thanks!

```
A problem occurred configuring root project 'app'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve com.osacky.doctor:doctor-plugin:0.7.2.
     Required by:
         project :
      > No matching variant of com.osacky.doctor:doctor-plugin:0.7.2 was found. The consumer was configured to find a runtime of a library compatible with Java 8, packaged as a jar, and its dependencies declared externally but:
          - Variant 'apiElements' capability com.osacky.doctor:doctor-plugin:0.7.2 declares a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares an API of a component compatible with Java 11 and the consumer needed a runtime of a component compatible with Java 8
          - Variant 'runtimeElements' capability com.osacky.doctor:doctor-plugin:0.7.2 declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 8
          - Variant 'testFixturesApiElements' capability com.osacky.doctor:doctor-plugin-test-fixtures:0.7.2 declares a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares an API of a component compatible with Java 11 and the consumer needed a runtime of a component compatible with Java 8
          - Variant 'testFixturesRuntimeElements' capability com.osacky.doctor:doctor-plugin-test-fixtures:0.7.2 declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 
```

**Comment by kelvinharron** [comment]
Huge thanks for the quick turnaround on this!

**Comment by runningcode** [comment]
Forgot to mention here that it was released in 0.7.3. https://runningcode.github.io/gradle-doctor/changelog/#073
